analysis/test/Cargo.toml

```diff
@@ -7,3 +7,4 @@ edition = "2021"
 
 [dependencies]
 libc = "0.2"
+c2rust-analysis-rt = { path = "../../analysis/runtime"}
```

Comment:
One idea I had was to make it an optional dependency, and then we can only enable that feature when instrumenting.  I think that's definitely preferable to the current solution.  We could also `cargo add` that dependency ourselves so that the user doesn't have to do it manually.  Another solution I was thinking of was to copy the existing `Cargo.toml`, add the `c2rust-analysis-rt` dependency to the copy, and then specify it with `--manifest-path` (I don't see an environment variable for that), though not quite all `cargo` commands allow `--manifest-path` (though I think all the ones we care about do).

Reply:
It's now optional in https://github.com/immunant/c2rust/pull/554/commits/de3fd2a1aa1b9b1970871f8aefd556831faf25a1.

@rinon, let me know if you want me to implement the other suggestions:
* automatically `cargo add /path/to/runtime --optional`, but otherwise the same
* create a shadow copy of the `Cargo.toml` at `Cargo.instrument.toml`, `cargo add /path/to/runtime` to it, run using `--manifest-path Cargo.instrument.toml`, and then delete it

Reply:
The `cargo add` approach is done in https://github.com/immunant/c2rust/pull/562, but is blocked on https://github.com/immunant/c2rust/pull/513, which upgrades our pinned `nightly` to one that supports `cargo add` natively.